.circleci/config.yml

```diff
@@ -0,0 +1,59 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    working_directory: ~/workdir
+    steps:
+      - checkout
+      - run: 
+          name: Clone test project
+          working_directory: ~/
+          command: |
+            (git -C ~/codacy-plugins-test fetch --all && 
+              git -C ~/codacy-plugins-test reset --hard origin/master) || 
+                git clone git://github.com/codacy/codacy-plugins-test.git ~/codacy-plugins-test
+      - restore_cache:
+          key: dependencies-{{ checksum "~/codacy-plugins-test/build.sbt" }}
+      - run: 
+          name: Compile test project
+          working_directory: ~/codacy-plugins-test
+          command: sbt compile
+      - run: 
+          name: Publish tool docker locally
+          working_directory: ~/workdir
+          command: docker build -t ${CIRCLE_PROJECT_REPONAME}:latest .
+      - save_cache:
+          key: dependencies-{{ checksum "~/codacy-plugins-test/build.sbt" }}
+          paths:
+            - "~/.ivy2"
+            - "~/.m2"
+            - "~/.sbt"
+            - "~/codacy-plugins-test/target"
+            - "~/codacy-plugins-test/project/target"
+            - "~/codacy-plugins-test/project/project"
+            - "~/workdir/target"
+            - "~/workdir/project/target"
+            - "~/workdir/project/project"
+      - run: 
+          name: Test json
+          working_directory: ~/codacy-plugins-test
+          command: sbt -Dcodacy.tests.ignore.descriptions=true "runMain codacy.plugins.DockerTest json $CIRCLE_PROJECT_REPONAME:latest"
+      - run:
+          name: Test patterns
+          working_directory: ~/codacy-plugins-test
+          command: sbt -Dcodacy.tests.noremove=true -Dcodacy.tests.threads=8 "runMain codacy.plugins.DockerTest pattern $CIRCLE_PROJECT_REPONAME:latest"
+      - deploy:
+          name: Push application Docker image
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              docker login -u $DOCKER_USER -p $DOCKER_PASS
+              docker tag ${CIRCLE_PROJECT_REPONAME} codacy/${CIRCLE_PROJECT_REPONAME}
+              docker tag ${CIRCLE_PROJECT_REPONAME} codacy/${CIRCLE_PROJECT_REPONAME}:1.0.$CIRCLE_BUILD_NUM
+              docker push codacy/$CIRCLE_PROJECT_REPONAME
```

Comment:
Can you push specifically `codacy/${CIRCLE_PROJECT_REPONAME}:1.0.$CIRCLE_BUILD_NUM` and `codacy/${CIRCLE_PROJECT_REPONAME}:latest` instead of all the tags?

Reply:
Yes, that's safer. I'm also going to rebase on current master to check that is passing on CircleCI.